hushline/model.py

```diff
@@ -28,6 +28,14 @@ def default(cls) -> "SMTPEncryption":
         return cls.StartTLS
 
 
+class HostOrganization(Model):
+    __tablename__ = "host_organization"
+
+    id: Mapped[int] = mapped_column(primary_key=True, default=1)
```

Comment:
Primary keys shouldn't have a default. Also if we're expecting some "main org" to have a certain key, then we should make that a constant within the class so we can reference it elsewhere.

Also the two other fields shouldn't have defaults either, I don't think. The default should be at the class level or in templates otherwise changing the defaults will require Alembic migrations.

Reply:
The primary key was given a default so the application logic wouldn't be dependent on an implementation detail of how a given database system allocates primary keys (start at 0 or 1 or other?, randomized or incrementing?). I'd rather that defaults lower such cognitive burdens. However, I've moved the intended behavior to a new ``__init__()`` to avoid the strangeness of a default primary key.

There's only so much DRY that can be achieved with ease or without problems. For instance, the CSS hard encodes the default primary color here

https://github.com/scidsg/hushline/blob/2cf9518c8cac279d9f03b1ee1b4b9bf8e5148295/hushline/static/css/style.css#L5

which shouldn't be removed, because it's an independent file that should be self-contained & shouldn't break if not connected with the rest of the app infrastructure.

Instead of having the styles produce nulls as a measure to detect incoherent states, I'd rather there be actual runtime (or even better compile time) errors. Partially with that in mind, I've structured this table so that the whole organization is represented by a single record. It either exists or doesn't. If it doesn't, then it is initialized to a coherent state. If it is, then only validated values are allowed to change the already coherent state, which should keep it in a coherent state. By making each attribute a row, it becomes a cognitive burden to ensure when accessing a record that all the other associated records are also defined. It's valuable to remove potentials for inconsistency.

Addressed by (https://github.com/scidsg/hushline/pull/573/commits/007cf1b1688684d8cf8751d28d8f4537788fb390)